scipy/signal/windows/_windows.py

```diff
@@ -128,8 +128,7 @@ def boxcar(M, sym=True):
     Parameters
     ----------
     M : int
-        Number of points in the output window. If zero or less, an empty
-        array is returned.
+        Number of points in the output window. If zero an empty array is returned.
```

Comment:
I think that returning an empty array if zero is provided is not so important. From a user standpoint, an exception thrown when it is negative is mote important information, it should be described.

Reply:
fixed